src/main/java/tc/oc/tablist/TabRender.java

```diff
@@ -53,6 +54,9 @@ private BaseComponent getContent(TabEntry entry, int index) {
   }
 
   private void appendAddition(TabEntry entry, int index) {
+    if (ViaUtils.getProtocolVersion(this.view.getViewer()) < 47) {
```

Comment:
This is a complete and absolute hack. Don't change the current tab view to avoid sending packets to 1.7 players, give them a new no-op view